apps/backend/src/api/export_import.py

```
@@ -0,0 +1,785 @@
+                    bytes_per_second = int(export_op.archive_size_bytes / elapsed_time_seconds)
+
+        return ExportProgressResponse(
+            export_id=export_id,
+            status=export_op.status,
+            progress_percentage=progress_percentage,
+            total_snapshots=export_op.total_snapshots,
+            processed_snapshots=export_op.exported_snapshots + export_op.failed_snapshots,
+            exported_snapshots=export_op.exported_snapshots,
+            failed_snapshots=export_op.failed_snapshots,
+            skipped_snapshots=export_op.skipped_snapshots,
+            current_operation=f"Exporting configurations"
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)